docs/features/feature-guides/compliance-forms/create-a-form.md

```diff
@@ -0,0 +1,159 @@
+    * _E.g. What types of personally identifiable information (PII) are included in this Asset? Email, Address, SSN, etc._
+* **Structured Properties:** Apply custom properties to an Asset and/or Column.
+    * _E.g. What date will this Dataset be deprecated and deleted?_
+
+When creating a Question, be sure to give it an easy-to-understand Title, and provide additional context or direction in the Description.
+
+<p align="center">
+  <img
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-prompt.png"
+       alt="Create a new Compliance Form prompt"/>
+</p>
+
+### Step 3: Assign your Form to relevant Assets
+
+Now that you have defined the Questions you want Users to complete, it's now time to select the in-scope Assets for this exercise.
+
+In the **Assign Assets** section, you can easily target the specific set of Assets that are relevant for this Form with the following steps:
+
+1. Add a Condition or Group of Conditions
+2. Choose the appropriate filter type, such as:
+    * Asset Type (Dataset, Chart, etc.)
+    * Platform (Snowflake, dbt, etc.)
+    * Domain (Sales, Marketing, Finance, etc.)
+    * Assigned Owners
+    * Assigned Glossary Terms
+3. Decide between **All**, **Any**, or **None** of the filters should apply
+4. Preview the relevant Assets to confirm you have applied the appropriate filters
+
+<p align="center">
+  <img 
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-assign-assets.png"
+       alt="Assign assets to a Compliance Form"/>
+</p>
+
+### Step 4: Add Recipients to your Form
+
+Now that you have defined the set of Questions to be answered for a set of Assets, it's now time to delegate out to your Users.
+
+In the **Add Recipients** section, decide who is responsible for completing the Form:
+
+* **Asset Owners:** Any User that is assigned to one of the in-scope Assets will be able to complete the Form. This is useful for larger initiatives when you may not know the full set of Users.
+* **Specific Users and/or Groups:** Select a specific set of Users and/or Groups within DataHub. This is useful when Ownership of the Assets may be poorly-defined.
+
+<p align="center">
+  <img 
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-add-users-or-groups.png"
+       alt="Assign recipients to a Compliance Form"/>
+</p>
+
+### Step 5: Publish your Form
+
+Once you have defined the set of Questions to be completed, the in-scope Assets, and the relevant Recipients, click **Publish** and Users will be able to complete your Form!
```

Comment:
Once you have defined the set of Questions to be completed, assigned Assets, and added recipients, you are ready to publish your Form. Once published, assignees will be notified to complete the form for the selected assets they are responsible for. 

To publish a Form, simply click **Publish**.

Reply:
(Suggestion ^)